BLSQ/iaso
# POLIO-945: remove Detection tab

The fields of this tab are already available in the Base info tab and The “Responsible field” should be removed too so the whole Detection tab can be removed.

Related JIRA tickets : [POLIO-945](https://bluesquare.atlassian.net/browse/POLIO-945?atlOrigin=eyJpIjoiNTYzZDAyNTQ5NjRkNDcxMGI1N2ViNWRhNTVhZDg5NGMiLCJwIjoiaiJ9)

## Self proofreading checklist


## Changes

- delete the obselete Detection form
- remove the Detection tab from `CreateEditDialog`

## How to test

- Go to Campaigns
- Create or Edit a campaign
- Detection tab should be removed

## Print screen / video

![Capture d’écran 2023-04-12 à 13 30 07](https://user-images.githubusercontent.com/25134301/231444310-f38bb0da-d363-4879-84f6-ec9b73dd9140.png)


[POLIO-945]: https://bluesquare.atlassian.net/browse/POLIO-945?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
